sympy/sets/sets.py

```diff
@@ -278,10 +278,10 @@ def contains(self, other):
 
         """
         other = sympify(other, strict=True)
-        ret = self._contains(other)
+        ret = sympify(self._contains(other))
```

Comment:
> This is the line which behaves differently because of sympify

Yes this seems true. Thanks for figuring out.
> currently -oo is considered real and reals are considered complex.

I think this problem is "not what is considered is `real` or `complex`". But the problem is with `SymPy bool` and `Python bool`
```        ### Before my changes
>>> S.Complexes.contains(S.Reals.start)
True               # python bool
```

```          # After i made my changes
>>> S.Complexes.contains(S.Reals.start)
true                # SymPy bool
```